{{cookiecutter.project_slug}}/justfile

```
@@ -0,0 +1,33 @@
+export COMPOSE_FILE := "docker-compose.local.yml"
+
+# Default command to list all available commands.
+default:
+    @just --list
+
+# build: Build python image.
+build:
+    @echo "Building python image..."
+    @docker-compose build
```

[We moved](https://github.com/cookiecutter/cookiecutter-django/pull/4463) to `docker compose` a while back:

```suggestion
    @docker compose build
```

The standalone `docker-compose` started to break as the compose Docker pluging gained adoption: https://github.com/cookiecutter/cookiecutter-django/issues/3980

---

I see. I have updated it to use `docker compose`